pcsx2/Patch.cpp

```diff
@@ -33,7 +34,8 @@
 extern void _ApplyPatch(IniPatch *p);
 
 
-IniPatch Patch[ MAX_PATCH ];
+
+std::vector<IniPatch> Patch;
```

Comment:
You should watch out for all uses of the 'patchnumber' variable. I admit that I don't fully understand its uses (for example I have no idea why 'patchnumber' is used at all in _LoadPatchFiles() since that function doesn't call any other function that modifies this variable). Perhaps all uses of 'patchnumber' can be cleaned up and the variable may either become unnecessary, or can be replaced with 'Patch.size()' where needed.

But one place where you definitely missed something is ForgetLoadedPatches(). You should be clearing the vector there

Reply:
iniprocess->initable->PatchTableExecute which executes a different function in a table called commands_patch depending if this is a comment or a patch. In our case it is a patch so PatchFunc::patch is called, which does this:
`	void patch(const wxString& cmd, const wxString& param) { patchHelper(cmd, param); }
`
which calls our patchHelper, trying to load patches :D
Altho darn, indeed missed it! Going to fix it now!

Reply:
Ah! That clears it up for me, thanks!